index.html

```diff
@@ -62,37 +62,47 @@ <h3 class="meal-name">
                     </div>
                 {{~}}
             </div>
-            
         </script>
     </head>
-    
+
     <body>
         <div class="top-container">
-            <h1>TTY:n ruokalistat</h1>
-            
+            <div>
+                <div id="tunilogo"></div>
```

Comment:
This is not accessible to visually impaired people. The logo should be added as a `img` tag with proper alt text

Reply:
Good point that I thought of. But decided that the easier implementation of hover effect with css vs javascript execution wins over the need for alt img. It does say 'Roukalistat' which should be enough information.